ROADMAP.md

```diff
@@ -54,14 +54,18 @@ _This section is still a very high-level draft_
 Testground As A Service embodies our long-term vision.
 
 - A single, scalable, installation that one or more organizations can use in all their projects,
+- An easy-to-follow documentation and guidelines for users to jump straight into action,
+  - An example: pages that can hand-hold through the learning curve, thus fostering community-driven tutorials. As well as APIs section that can help navigate through the vast possibilities of testground.
 - The ability to experiment with large-scale networks and simplify the integration testing of "any" network-related code,
+  - An example: experiment with a set of rollup networks that deploy their data to a different DA network. A unified/standardised way of doing network(s)-to-network(s) experiments would eliminate a lof of bike-shedding that future teams will go through. 
```

Comment:
let's drop the wording around `bike-shedding`, maybe:


```suggestion
  - An example: having the ability to define standard experiments with a set of roll-up networks that deploy their data to a different DA network. This would help teams compare solutions using the same metrics.
```